code/game/objects/items/melee/baton.dm

```diff
@@ -70,6 +74,10 @@
 
 	register_item_context()
 
+/obj/item/melee/baton/examine(mob/user)
+	. = ..()
+	. += span_notice("\The [src] has an armor penetrative power of [get_stun_penetration_value()]%.")
```

Comment:
Move this into combat tips maybe? If it doesn't have one already, make a comsig to display additional info there

Reply:
Adding combat information is handled via ``proc/add_weapon_description`` adding the element, so you can just override that to pass along an additional proc in the args to include extra information. We do it for ammo casings and stuff.

Reply:
I think that's a better option than sticking it into examine?